src/components/atoms/VisualisationSelector.tsx

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
 import { SelectField } from 'evergreen-ui';
+import { VisualisationType } from '../../types/Metadata';
 
 type Props = {
+    default: VisualisationType;
     label: string;
     options: Record<string, string>;
     onChange: React.ChangeEventHandler<HTMLSelectElement>;
 };
 
 const VisualisationSelector: React.FC<Props> = (props) => (
-    <SelectField label={props.label} defaultValue={Object.values(props.options)[0]} onChange={props.onChange}>
+    <SelectField label={props.label} value={props.default} onChange={props.onChange}>
```

Comment:
Why did you remove defaultValue?

Reply:
The default value would only work if you re-freshed the page or viaited it for the first time.